### DISCUSSION TITLE
O meu carregamento do comprovante do meu estatuto academico falha direto //  CM team translation:  My academic status proof upload fails directly

### DISCUSSION THREAD

**Discussion by abiga-m** [community/community#21975] [GitHub Education]
<p>Olá, boa tarde! Faz um bom tempo que minha foto e comprovativo do meu estatuto acadêmico está e nunca chega ao final, ademais não está aceitando pdf, gostaria de resolver carregando esse imbróglio o mais rápido possível, pois preciso urgentemente.</p>

**Comment by jfullstackdev** [comment]
<p>yup, I’m seeing here the same situation as yours, pending approval,</p>

**Reply by vitormendesvr** [reply]
não estou conseguindo enviar meu comprovante de escolaridade, tem dado erro, não tem nenhuma opção de anexar arquivo ao invés de ter que tirar foto?

**Reply by ettaboyle** [reply]
Hi @vitormendesvr  👋 Unfortunately, we don’t currently have moderators for languages other than English. What this means is that, until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We're experimenting with in-product translation, but as a best practice, if you can post in English you'll likely receive a response faster.

Thanks and happy to have you here ✨

**Reply by vitormendesvr** [reply]
ok, i did it, thanks

**Comment by queenofcorgis** [comment, marked as answer]
<p>Hi there <a class="mention" href="/abiga-m">@abiga-m</a> :wave:  For student benefits and related conversations, we recommend using our <a href="https://github.com/orgs/github-community/discussions/categories/github-education">Education Discussion</a> as you’ll have better visibility among those who may be able to answer your question.</p>

**Comment by vitormendesvr** [comment]
I'm not able to send my schooling attachment, it's been giving an error, there's no file option instead of having to take a photo?
